main/zoe/api/zoe-contract-facet.md

```diff
@@ -293,26 +320,8 @@ to manipulate the offer. The queries and operations are as follows:
      ```js
      throw seat.fail(Error('you did it wrong'));
      ```
-### `ZCFSeat.stage(newAllocation)`
-   - `newAllocation`: `{Allocation}`
-   - Returns: `{SeatStaging}`
-   - An `Allocation` is an `AmountKeywordRecord` of key-value pairs where
-     the key is a keyword such as `Asset` or `Price` applicable to the
-     contract. The value is an `amount` with its `value` and `brand`.
-    
-     A `seatStaging` is an association of a `seat` with reallocations. `reallocate()` takes
-     at least two `seatStagings` as arguments and does its reallocation based on them.
-
-     You can create multiple independent `seatStagings` for a `seat`. None of them has
-     any effect until submitted to `reallocate()`. Each call to `stage()` starts from the
-     `seat`'s current `allocation` and uses the `newAllocation` as a replacement for the
-     current state. Any keywords not mentioned in `newAllocation` retain the
-     same `amounts`. All keywords mentioned in the `newAllocation` have their `amounts`
-     replaced with the corresponding `amount` from `newAllocation`.
-
-     Note that ZoeHelpers [`trade()`](./zoe-helpers.md#trade-zcf-left-right-lefthasexitedmsg-righthasexitedmsg) 
-     and [`swap()`](./zoe-helpers.md#swap-zcf-leftseat-rightseat-lefthasexitedmsg-righthasexitedmsg) might
-     be easier to use for simple cases.
+### `ZCFSeat.stage(newAllocation)`**DEPRECATED 22-06-01**
```

Comment:
I think it's going to be *gone*, not *deprecated*

Reply:
That's a good point. We should probably distinguish between methods that exist and are deprecated, and methods that are gone. I think at this point, we should remove anything from documentation that is removed in the code.

Reply:
Although at some point, we do want to start leaving indicators in the docs for a while when we drop something, since someone will probably have been using it.

Reply:
Yup, agreed. I think we should start that after we finish this string of known breaking changes for Zoe and ERTP.